addons/packages/fluxcd-helm-controller/0.17.2/package.yaml

```
@@ -0,0 +1,39 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: Package
+metadata:
+  name: helm-controller.fluxcd.community.tanzu.vmware.com.0.17.2
+spec:
+  refName: helm-controller.fluxcd.community.tanzu.vmware.com
+  version: 0.17.2
+  releasedAt: "2022-05-01T16:40:55"
```

```suggestion
  releasedAt: 2022-05-01T16:40:55Z
```
@hemakshis Last change, I promise! The `releasedAt` parameter is a little touchy with the format. If the time format is not specified like in this suggestion, then it will error when the package repo is installed to the the cluster.

This is what the error looks like:
```
 Package in version "v1alpha1" cannot be handled as a Package: v1alpha1.Package.Spec: v1alpha1.PackageSpec.Template: v1alpha1.AppTemplateSpec.ReleasedAt: unmarshalerDecoder: parsing time "2022-05-01T16:40:55" as "2006-01-02T15:04:05Z07:00": cannot parse "" as "Z07:00", error found in #10 byte of ...|T16:40:55","template|..., bigger context ...|nzu.vmware.com","releasedAt":"2022-05-01T16:40:55","template":{"spec":{"deploy":[{"kapp":{}}],"fetch|... (reason: BadRequest). Reconcile failed: Deploying: Error (see .status.usefulErrorMessage for details)
```

---

fixed it.